.prettierrc

```diff
@@ -1,9 +1,18 @@
 {
-  "trailingComma": "none",
-  "tabWidth": 4,
-  "semi": true,
-  "singleQuote": false,
-  "arrowParens": "always",
-  "printWidth": 80,
-  "useTabs": false
+    "trailingComma": "none",
+    "tabWidth": 4,
+    "semi": true,
+    "singleQuote": false,
+    "arrowParens": "always",
+    "printWidth": 80,
+    "useTabs": false,
+
+    "plugins": [
+        "prettier-plugin-tailwindcss",
+        "@ianvs/prettier-plugin-sort-imports"
+    ],
+
+    "tailwindFunctions": ["cva"],
```

Comment:
nice addition but we'll be getting rid of cva in #459

Reply:
I saw that but thought it could be here until the rewrite is done, but I did forget `classList`